client/common/cmdline.c

```diff
@@ -2289,6 +2295,49 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 			if (!(settings->RemoteAssistancePassword = _strdup(arg->Value)))
 				return COMMAND_LINE_ERROR_MEMORY;
 		}
+		CommandLineSwitchCase(arg, "pwidth")
+		{
+			settings->DesktopPhysicalWidth = atoi(arg->Value);
+		}
+		CommandLineSwitchCase(arg, "pheight")
+		{
+			settings->DesktopPhysicalHeight = atoi(arg->Value);
+		}
+		CommandLineSwitchCase(arg, "orientation")
+		{
+			settings->DesktopOrientation = atoi(arg->Value);
+		}
+		CommandLineSwitchCase(arg, "scale")
+		{
+			int scaleFactor = atoi(arg->Value);
+			if (scaleFactor == 100 || scaleFactor == 140 || scaleFactor == 180) {
```

Comment:
@colemickens Nitpicking but freerdp's coding style puts { on its own line (i.e. see the CommandLineSwitchCase lines):
```
}
CommandLineSwitchCase(arg, "")
{
```